### PR TITLE
Add Documentation for ECHO command #872

### LINKED ISSUE #872: Add Documentation for "ECHO" command
Right now we don't have documentation of "ECHO" command, command is already being implemented as part of https://github.com/DiceDB/dice/issues/622.

### PR DESCRIPTION
**Pull request by mohdammar128**
### Summary of Changes
This pull request adds documentation for the `ECHO` command in DiceDB. 

### Changes Made
- Added details about the syntax and parameters of the `ECHO` command.
- Included examples of valid and invalid usage.
- Updated the documentation to clarify error messages and return values.

### Related Issues
Fixes #872

### How to Test
To test the `ECHO` command, you can run the following command in the DiceDB terminal:

### CODE REVIEW

File: docs/src/content/docs/commands/ECHO.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+title: ECHO
+description: `ECHO` command in DICEDB is used to print a message
```
Comment (lucifercr07) [inline docs/src/content/docs/commands/ECHO.md:3]: `DICEDB` -> `DiceDB`

File: docs/src/content/docs/commands/ECHO.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+```
+ECHO message
+
+```
+## Parameters
+
+| Parameter | Description                                                               | Type    | Required |
+|-----------|---------------------------------------------------------------------------|---------|----------|
+| `message` | A string of characters or numbers to print                                | String  | Yes      |
```
Comment (lucifercr07) [inline docs/src/content/docs/commands/ECHO.md:16]: `message` can be of both type `String/Number` correct?

**Review by lucifercr07 — CHANGES_REQUESTED** [review]

File: docs/src/content/docs/commands/ECHO.md (hunk 2 above)
Comment (mohdammar128) [reply docs/src/content/docs/commands/ECHO.md:16]: Yes

File: docs/src/content/docs/commands/ECHO.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+### Invalid Usage
+
+```bash
+127.0.0.1:7379> ECHO 
+```
+`Response`:
+- `(error) ERROR wrong number of arguments for 'echo' command`
+
+```bash
+127.0.0.1:7379> ECHO "DiceDB is" "very effcinet"
```
Comment (preetmungara) [inline docs/src/content/docs/commands/ECHO.md:50]: fix the typo to "efficient".

Comment (mohdammar128) [reply docs/src/content/docs/commands/ECHO.md]: Done

File: docs/src/content/docs/commands/ECHO.md (hunk 1 above)
Comment (mohdammar128) [reply docs/src/content/docs/commands/ECHO.md]: Done

**Comment by mohdammar128** [comment]
can you please review it again ,I have made changes as mentioned ? Please let me know if any changes is required.
@lucifercr07 
@preetmungara

**Review by lucifercr07 — APPROVED** [review]

**Comment by rishavvajpayee** [comment]
hey @lucifercr07 i got build fails due to `ECHO` in the description. was working on SDIFF documentation. Will be doing this change in my PR though

**Pull request merged**